### ISSUE TITLE
API versioning

### ISSUE THREAD

**Issue by isabelcosta** [anitab-org/mentorship-backend#481]
## Description

> Add versioning to backend API, so endpoints URL contains v1 as a part of the path. Make sure we can keep backward compatibility, so endpoints determinate which code is called for v1 and next versions (keeping in different folders or adding parameter or any other idea).

initially brought up by @annabauza 

Before implementing this I think it's important to think about this solution, so before submitting a PR for this, please let us know what you intend to do for this.

Flask RESTplus allows using Blueprints: https://flask-restplus.readthedocs.io/en/stable/scaling.html#use-with-blueprints

### Update [Required]
- [ ] Come with a solution to add API versioning to backend

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours

**Comment by pallavisavant** [comment]
I have participated in OSH can i take up this ,if its available.Thanks.

**Comment by SanketDG** [comment]
This issue is not part of OSH, but if you are interested enough and want to take a stab at this, we can add the label definitely.

**Comment by pallavisavant** [comment]
@SanketDG yes i would like to try...lemme see i will do some more research and get back here...Thanks.

**Comment by pallavisavant** [comment]
> This issue is not part of OSH, but if you are interested enough and want to take a stab at this, we can add the label definitely.

I would like to work on this as part of OSH...pls assign it to me ....Thanks.

**Comment by vj-codes** [comment]
@pallavisavant any updates?

**Comment by pallavisavant** [comment]
> @pallavisavant any updates?

I have unassigned myself,pls feel free to assign it to anyone else,sorry for inconvenience I had got busy with my college work:)

**Comment by sanjibansg** [comment]
Do assign the issue to me if available. @vj-codes

**Comment by vj-codes** [comment]
Assigning you @kahanikaar 
Happy researching!

**Comment by devkapilbansal** [comment]
@kahanikaar what's the status of this issue??

**Comment by devkapilbansal** [comment]
@vj-codes I think that this issue is available now as the assignee is not responding. Please update the labels accordingly.

**Comment by vj-codes** [comment]
Removing your assignment @kahanikaar due to inactivity .
